storage/src/lib.rs

```diff
@@ -1,2 +1,14 @@
+//! Storage for a node on the Casper network.
+
+#![doc(html_root_url = "https://docs.rs/casper-storage/1.4.3")]
+#![doc(
+    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
+    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    test(attr(forbid(warnings)))
+)]
+// TODO - enable and fix warnings.
+// #![warn(missing_docs)]
```

Comment:
nit: remove this comments

Reply:
Done in [b4bf2cf](https://github.com/casper-network/casper-node/pull/4101/commits/b4bf2cf0f94406f45caab2b6283325da51fb7e78) and a ticket request for this made.